src/Client.php

```diff
@@ -122,11 +122,10 @@ public function configure(array $options = array())
             $this->host = $options['host'];
         }
         if (isset($options['port'])) {
-            $port = (int) $options['port'];
-            if (! $port || !is_numeric($port) || $port > 65535) {
+            if (!is_numeric($options['port']) || $options['port'] < 0 || $options['port'] > 65535) {
```

Comment:
Could you please elaborate the requirement for port 0? It's not an official port and I'm slightly reluctant to modify the library to support it.

Secondly, although this now allows port 0, it also now allows `"1234"` (String) and decimals such as `1.23` which are invalid values for a port. Very unlikely for someone to try and use that, but I'd like to prevent it if possible via something like misconfigured ENV variables.

The previous behaviour was to force `$port` to always be an Integer type.

Could you please amend the code to force Int type on the port to avoid this.

Reply:
For me it was an easy way to disable logging, a quick way to send stats to the void. Issue #41 also addresses the possibility to use it as a testing port.

Regarding invalid ports, the string ports were already covered in the testStringPort (which was passing, string ports are not valid), and for the decimal ports I added another check with its corresponding unit test.

I had to remove the int casting, because if you cast a string into an int, you get 0, making all string ports valid (with this change of course)

I understand if you are reluctant to merge such change, feel free to reject it. No hard feelings :D 
If you think this could be useful but still have some doubts, maybe we can add a notice when using port 0 to warn the user.